File: spotify-docker-maven-plugin-example/src/main/java/de/marcusjanke/examples/springboot/SampleApp.java
package de.marcusjanke.examples.springboot;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@SpringBootApplication
@RestController
public class SampleApp {

    public static void main(String[] args) {
        SpringApplication.run(SampleApp.class, args);
    }

    @RequestMapping("/")
    public String index() {
        return "This is your spotify docker-maven-plugin sample app!";
    }
}

File: spotify-docker-maven-plugin-example/src/main/kotlin/de/marcusjanke/examples/springboot/SampleApp.kt
package de.marcusjanke.examples.springboot

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@SpringBootApplication
@RestController
open class SampleApp {

    @RequestMapping("/")
    fun index(): String {
        return "This is your spotify docker-maven-plugin sample app!"
    }
}

fun main(args: Array<String>) {
    runApplication<SampleApp>(*args)
}
